tests/functional/disaster-recovery/sc_arbiter/test_add_capacity.py

```diff
@@ -123,6 +135,29 @@ def test_cluster_expansion(
         sc_obj.post_failure_checks(start_time, end_time, wait_for_read_completion=False)
         logger.info("Successfully verified with post failure checks for the workloads")
 
+        # check vm data written before the failure for integrity
```

Comment:
this md5sum is captured after the failure. Please update the comment accordingly